docs/source/daml/reference/interfaces.rst

```diff
@@ -18,6 +18,19 @@ method) are implemented. This allows decoupling such behavior from its
 implementation, so other developers can write applications in terms of the
 interface instead of the concrete template.
 
+Configuration
+*************
+In order to be able to test this experimental feature, you need to enable
+the dev mode on both Daml and Canton.
+
+For Daml, you need to add in ``daml.yaml`` the following line:
```

Comment:
```suggestion
To enable dev mode for Daml, add ``daml.yaml`` the following line:
```
Perhaps this is clear to a more experienced programmer, but for completeness' sake: do you add daml.yaml to the end of the line, or where exactly?

Reply:
So what I meant is that : there is an existing file named `daml.yaml` and this file we need a code line.